Assets/Scripts/Card/GrenadeThrow.asset

```diff
@@ -0,0 +1,16 @@
+%YAML 1.1
```

Comment:
asset w skrypcie?

Reply:
Hmm. No mogę to zmienić. Tylko to jest scriptableobject jak go się stworzy z menu... To nie za bardzo wiem gdzie go trzymać